merkletree/pad.go

```diff
@@ -21,12 +21,12 @@ type PAD struct {
 	snapshots    map[uint64]*SignedTreeRoot
 	loadedEpochs []uint64 // slice of epochs in snapshots
 	latestSTR    *SignedTreeRoot
-	policies     Policies // the current policies in place
+	policies     *Policies // the current policies in place
```

Comment:
> pad.policies is in use, so the same mutation comment applies here.

Yup, that's why I have this bug bc36c45e36c9eb51bc7510034ac43f80c44f3b30 (fixed in this commit). Thanks!

> I think you should revert everything in this patch that isn't strictly about removing the interface, sorry.

Oops, my bad. I split it into a separate commit (af994216261da5c00278929e1638e3ca682d4247)

Btw, I'm so sorry because of these late commits and response.